lms/templates/course_modes/track_selection.html

```diff
@@ -0,0 +1,207 @@
+                title.attr("aria-expanded", "true");
+            } else {
+                title.attr("aria-expanded", "false");
+            }
+        };
+
+        $(document).ready(function() {
+            $('.expandable-area').slideUp();
+            $('.is-expandable').addClass('is-ready');
+
+            $('.is-expandable .title-expand').click(expandCallback);
+            $('.is-expandable .title-expand').keypress(function(e) {
+                if (e.which == 13) {  // only activate on pressing enter
+                    expandCallback.call(this, e); // make sure that we bind `this` correctly
+                }
+            });
+
+            $('#contribution-other-amt').focus(function() {
+                $('#contribution-other').attr('checked',true);
+            });
+
+            % if use_ecommerce_payment_flow:
+            $('button[name=verified_mode]').click(function(e){
+                e.preventDefault();
+                window.location.href = '${ecommerce_payment_page | n, js_escaped_string}?sku=' +
+                encodeURIComponent('${sku | n, js_escaped_string}');
+            });
+            % endif
+        });
+
+        // popover icon toggle
+        $(function(){
+            $('body').click(function (e) {
+                e.stopPropagation();
+                $('.popover').css({"visibility": "hidden", "opacity": 0});
+            });
+            $('.popover-icon').click(function(e){
+                e.stopPropagation();
+                if ($('.popover').css("visibility") == "hidden" || $('.popover').css("visibility") == "" ) {
+                    $('.popover').css({"visibility":"visible", "opacity": 1});   
+                } else {
+                    $('.popover').css({"visibility":"hidden", "opacity": 0});   
+                }
+            });
+        });
+
+        // resize window
+        var onresize = function(e) {
+            width = e.target.outerWidth;
+            if(width < 576) {
+                $('.collapsible-item').css({"display":"none"});
+                $('.collapsible').css({"display":"block"});
+            };
+        }
+        window.addEventListener("resize", onresize);
+
+        // responsive: show more 
+        $(function(){
+            if($(window).width() < 576) {
+            $('.collapsible-item').css({"display":"none"});
+            $('.collapsible').css({"display":"block"});
+            };
+        });
+        $(function(){
+            $('.collapsible').click(function(){
+                $('.collapsible').css({"display":"none"});
+            $('.collapsible-item').css({"display":"list-item"});
+            }); 
+        });
+    </script>
+</%block>
+<%static:webpack entry="Currency">
+    new Currency();
+</%static:webpack>
+
+<%block name="header_extras">
+    <link rel="stylesheet" type="text/css" href="${static.url('paragon/static/paragon.min.css')}" />
+</%block>
+
+<%block name="content">
+    % if error:
+        <div class="wrapper-msg wrapper-msg-error">
```

Comment:
This is the current page's error message - working with UX to determine what the new error message handling will be. TBD if it will be added in this ticket.

Reply:
This would be considered non-happy path, removed in second commit